UnoCheck/Solutions/GTK3NotInstalledOnWindowsSolution.cs

```diff
@@ -0,0 +1,52 @@
+﻿using DotNetCheck.Models;
+using System;
+using System.IO;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace DotNetCheck.Solutions
+{
+	public class GTK3NotInstalledOnWindowsSolution : Solution
+	{
+		public override async Task Implement(SharedState sharedState, CancellationToken cancellationToken)
+		{
+			byte[] data = null;
+
+			WebClient client = new();
```

Comment:
There's already a solution that exists which does something very similar. Reuse or adjust he code from:

https://github.com/unoplatform/uno.check/blob/1aa1a2c0df19c860b71c68b0d3b356cc21800e15/UnoCheck/Solutions/MsInstallerSolution.cs#L42

Reply:
Done!